CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 - Minor: Added icons for newer versions of macOS. (#5148)
 - Minor: Added the `--incognito/--no-incognito` options to the `/openurl` command, allowing you to override the "Open links in incognito/private mode" setting. (#5149)
 - Minor: Added support for the `{input.text}` placeholder in the **Split** -> **Run a command** hotkey. (#5130)
+- Minor: Added the ability to pin a popup window. (#5135)
```

Comment:
Should be updated to reflect the phrasing "top-most", and that it can be used to un-top-most a window. I don't know how to phrase it :1st_place_medal: . PR title should be the same as this entry basically